### PR TITLE
docs: update multi select dialog api

### PR DESCRIPTION
**Pull request by nextchamp-saqib**
Docs for - https://github.com/frappe/frappe/pull/12471
---

Screenshot 2021-05-13 at 5 06 46 PM

### CODE REVIEW

File: frappe_docs/www/docs/user/en/api/dialog.md [hunk 1]
```diff
@@ -347,3 +352,38 @@ new frappe.ui.form.MultiSelectDialog({
...
+
+```js
+
+// MultiSelectDialog for individual child selection
+new frappe.ui.form.MultiSelectDialog({
+	doctype: "Material Request",
+	target: this.cur_frm,
+	setters: {
+		schedule_date: undefined,
+		status: undefined
```
Comment (surajshetty3416) [inline frappe_docs/www/docs/user/en/api/dialog.md:366]:
```suggestion
		schedule_date: null,
		status: null
```
The user-defined value should not be undefined

File: frappe_docs/www/docs/user/en/api/dialog.md [hunk 2]
```diff
@@ -329,8 +332,10 @@ new frappe.ui.form.MultiSelectDialog({
 	doctype: "Material Request",
 	target: this.cur_frm,
 	setters: {
-		company: "Zoot"
+		schedule_date: undefined,
```
Comment (nextchamp-saqib) [inline frappe_docs/www/docs/user/en/api/dialog.md:335]:
```suggestion
		schedule_date: null,
```

File: frappe_docs/www/docs/user/en/api/dialog.md [hunk 3]
```diff
@@ -306,8 +307,10 @@ new frappe.ui.form.MultiSelectDialog({
...
```
Comment (nextchamp-saqib) [inline frappe_docs/www/docs/user/en/api/dialog.md:310]:
```suggestion
		schedule_date: null,
```

File: frappe_docs/www/docs/user/en/api/dialog.md [hunk 4]
```diff
@@ -347,3 +352,38 @@ new frappe.ui.form.MultiSelectDialog({
...
+new frappe.ui.form.MultiSelectDialog({
+	doctype: "Material Request",
+	target: this.cur_frm,
+	setters: {
+		schedule_date: null,
+		status: null
+	},
+	add_filters_group: 1,
+	date_field: "transaction_date",
+	child_selection_mode: 1,
```
Comment (nextchamp-saqib) [inline frappe_docs/www/docs/user/en/api/dialog.md:370]:
```suggestion
	allow_child_item_selection: 1,
```

File: frappe_docs/www/docs/user/en/api/dialog.md [hunk 5]
```diff
@@ -347,3 +352,38 @@ new frappe.ui.form.MultiSelectDialog({
...
+	child_doctype: "Material Request Item", // the child item for filtering
```
Comment (nextchamp-saqib) [inline frappe_docs/www/docs/user/en/api/dialog.md:371]:
```suggestion
	child_fieldname: "items", // child table fieldname, whose records will be shown & can be filtered
```

**Comment by gavindsouza** [comment]
Closing since feature PR is now closed.

**Review by gavindsouza — APPROVED** [review]

**Pull request merged**